### PR TITLE
Issue/sample image fast

### PR DESCRIPTION
**Pull request by hauwenw**
改為 multiprocess 跑切圖 
4 core mac 半場可以從 28 分鐘加速到 18 分鐘

### CODE REVIEW

**Comment by lucemia** [comment]
good , how about on the server?

**Comment by lucemia** [comment]
@hauwenw  

multiprocessing will conflict with some package (such as gevent)
so please make a new method instead of replace current one

File: src/ffmpeg.py [hunk 1]
```diff
@@ -281,9 +281,10 @@ def _sample_images_fast(path, fps, opath):
         raise NotImplementedError
 
     duration = int(get_duration(path))
+    pool = multiprocessing.Pool()
```
Comment (lucemia) [inline src/ffmpeg.py]: Create new function instead of replace current one since multiprocessing will conflict with some package

**Comment by hauwenw** [comment]
should I merge before run on server?

**Pull request merged**